src/libs/ReportUtils.ts

```diff
@@ -514,6 +514,23 @@ Onyx.connect({
     },
 });
 
+let lastUpdatedReport: OnyxEntry<Report>;
+
+Onyx.connect({
+    key: ONYXKEYS.COLLECTION.REPORT,
```

Comment:
NAB but I think we're optimizing and use `waitForCollectionCallback: true` for all the report collections [here](https://github.com/Expensify/App/pull/38464), so I think it's good to has it here as well but if this PR can be merged before #38464 then I think I'm fine with this 😄

Reply:
should we address this given the linked PR was merged? @hungvu193

Reply:
I think Yes, cc @TMisiukiewicz

Reply:
sorry I missed that comment before. I've tried it and seems like it does not work properly with `waitForCollectionCallback`, I am not getting an update of the item here when it's set to `true`

Reply:
hmm yeah, I also got the issue, let skip it for now.